modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -11,14 +11,15 @@ import {Tileset3D, _getIonTilesetMetadata} from '@loaders.gl/3d-tiles';
 import {getFrameState} from './get-frame-state';
 
 const defaultProps = {
-  getColor: [255, 0, 0],
-  _lighting: 'pbr',
+  getPointColor: [255, 0, 0],
   pointSize: 1.0,
   opacity: 1.0,
 
-  tilesetUrl: null,
+  data: null,
   _ionAssetId: null,
   _ionAccessToken: null,
+  _loadOptions: {throttleRequests: true},
```

Comment:
Why underscore? This is an existing base layer prop and used by the `ScenegraphLayer`.